### PR TITLE
feat: export-poshtheme to clipboard if no file path provided

### PR DESCRIPTION
**Pull request by lnu**
### Prerequisites

- [x] I have read and understand the `CONTRIBUTING` guide
- [ ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Description

Export-PoshTheme to clipboard if no filepath provided

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: src/init/omp.ps1 [hunk 1]
```diff
@@ -104,25 +104,52 @@ function global:Write-PoshDebug {
...
+
+    # if no path, copy to clipboard by default
+    if ($FilePath -ne "") {
+        if ($FilePath.StartsWith('~')) {
+            $FilePath = $FilePath.Replace('~', $HOME)
+        }
+        [IO.File]::WriteAllLines($FilePath, $configString)
+    }
+    else {
+        set-clipboard $configString
```
Comment (JanDeDobbeleer) [inline src/init/omp.ps1]: Can we respect the casing? `Set-ClipBoard` and `Write-Output` ;-)

Comment (lnu) [reply src/init/omp.ps1]: yep

**Comment by lnu** [comment]
One more thing: I only tested it on windows. If you're on macos can you check that set-clipboard is working as expected? otherwise I have to reboot(I'm on bootcamp).

**Comment by JanDeDobbeleer** [comment]
@lnu works! If you rebase, it will merge ;-)

**Comment by lnu** [comment]
rebased

**Pull request merged**